views/partials/breach-groups.hbs

```diff
@@ -9,7 +9,7 @@
           <!-- foundBreaches 4-foundBreaches.length -->
           <div id="additional-breaches" class="show-additional-breaches hide flx">
             {{#each remainingBreaches }}
-              {{> breach-card cardType=./../cardType}}
+              {{> breach-card cardType=./../cardType LOGOS_ORIGIN=./../LOGOS_ORIGIN}}
```

Comment:
This doesn't seem to be the right path to the parent context with `LOGOS_ORIGIN` in it. :/ The dashboard breach groups still render without the logos origin value.